### DISCUSSION TITLE
Update Trilium on windows

### DISCUSSION THREAD

**Discussion by giuseppepagnoni** [TriliumNext/Trilium#3250] [Q&A] [answered]
I just downloaded the new zip file for the win64 version.
What is the correct way for updating my existing version without losing my current notes?
Thanks!

BTW: Trilium is great!!!

**Comment by zadam** [comment, marked as answer]
Just extract the ZIP into a folder of your choosing. Trilium stores its [data](https://github.com/zadam/trilium/wiki/Data-directory) in `C:\Users\[user]\AppData\Roaming\trilium-data`.
